README.md

```diff
@@ -2,34 +2,36 @@
 
 [![Build Status](https://travis-ci.org/redhat-developer/opencompose.svg?branch=master)](https://travis-ci.org/redhat-developer/opencompose)
 
-The goal of OpenCompose is to make it easier to on-board developers to Kubernetes.
-It is a curated higher level abstraction for specific Kubernetes resources.
-Very simple idea, isn't it? Developer shouldn't have to learn various Kubernetes concepts just to test and deploy their applications on Kubernetes.
-The developer is generally concerned with the application that is being developed.
+The goal of OpenCompose is to make it easier for developers to transition to Kubernetes.
```

Comment:
Definitely want "on-board" or anything that means "on-board". OpenCompose doesn't assume that people are using anything else before. It is supposed to be "kubernetes first".